qemu/tests/block_hotplug.py

```diff
@@ -210,7 +210,7 @@ def unplug_device(vm, get_disk_cmd, device):
             error_context.context(context_msg % (sub_type, "after hotplug"),
                                   logging.info)
             utils_test.run_virt_sub_test(test, params, env, sub_type)
-            if vm.is_dead():
+            if sub_type == "shutdown" and vm.is_dead():
```

Comment:
Hi coli, 
With this code, if also hit "guest did not boot up after sub test 'boot'", what's the error will report?
Thanks~

Reply:
If the guest is truly crash/dead, will raise VMDeadError which defined in qemu_vm.

Reply:
ok, thanks for your explanation, It's fine for me.